### DISCUSSION TITLE
Sàn UPCOM không có data price board

### DISCUSSION THREAD

**Discussion by thanhtie** [thinh-vu/vnstock#124] [Hỏi đáp (Q&A)] [answered]
em lên TCBS thì thấy trên đó vẫn list cổ phiếu UPCOM, nhưng khi em xài lệnh stock.trading.price_board([sicker]), sticker là một mã trên UPCOM thì báo lỗi, em thử các mã trên HSX thì bình thường, các mã trên UPCOM thì đều bị như vậy hết. Em k rõ mình có nhầm ở đâu không hay do thư viện chưa hỗ trợ, em xin cám ơn

**Comment by thinh-vu** [comment, marked as answer]
Hi em, anh đã kiểm tra và xác nhận có lỗi này với nguồn dữ liệu VCI nhé.
Anh sẽ sửa lỗi trong phiên bản tiếp theo. Trong thời gian này em có thể chuyển đổi thông tin `source='TCBS'` để sử dụng dữ liệu từ nguồn TCBS vẫn hoạt động bình thường.

**Reply by thanhtie** [reply]
em xin cám ơn
